src/common/Input.tsx

```diff
@@ -137,6 +156,30 @@ const Input: React.FC<InputProps> = (props) => {
           color: #d2d2d2;
           text-decoration: line-through;
         }
+        .password-input {
```

Comment:
Since you're only using this class for distinguishing whether the input is password, you can actually remove it and substitute it with:
```
input[type="password"] {
  // ...css
}
```

Reply:
As far as I can tell this doesn't work as is because this class is what I use here: `className={classnames({ "password-input": props.password })}`).
The class is applied when the input type is password according to the Input component props, not the html input element (which is what i'm changing back and forth to make the text visible).

Reply:
...and you'd be right! Sorry for misleading you!